### PR TITLE
Generate sch convex files / remove impossible collisions

### PR DESCRIPTION
**Pull request by arntanguy**
Today I noticed that the default collision ellipses read from the urdf are wholly inadequate and severely restrict the range of some joints.

This PR:
- generates and uses `sch` collision shapes generated directly from the `dae` meshes
- apply all object transformations to the dae models to prevent issues with various tools
- removes collision pairs that cannot happen (such as `link3 - link5`)

As a result we both get much much more precise collision avoidance, with the additional benefit of far less collision pairs active, which should more than offset the additional cost of computing the distances between sch shapes.

### CODE REVIEW

**Comment by gergondet** [comment]
If I recall well this was the original approach we used but it was problematic when running on the real robot because Franka Emika controller relies on the collision shapes defined in the urdf which are in fact quite restrictive compared to the real shapes of the robot.

Did you try this patch on the real
robot?

**Comment by arntanguy** [comment]
Ah, no i didn't have time to try it for real yet. I'll check, and if it proves problematic I'll see whether we can disable the default collision behavior. As-is it severely restrict some of the joint limits, so that's not ideal

**Comment by arntanguy** [comment]
You are right, this does not work on the real robot and I could not find a way to disable the default collision check behaviour.

Thus I've reverted to using the collision shapes defined in the URDF as our default minimal collision set.
I have however removed the following collisions that do not occur in practice:
- `panda_link3* - panda_link6*`
- `panda_link5* - panda_link7*`

At least this gives us back the full range for `panda_joint5` and does not trigger any collision error on the real robot.

Additionally, I've left the loading of the convex shapes but renamed as `convex_panda_link*`. These can still be usefull to check for robot-robot collisions (two pandas, or panda-hrp4).

File: src/panda.cpp [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 #include <RBDyn/parsers/urdf.h>
 
+#include <filesystem>
```
Comment (gergondet) [inline src/panda.cpp]: This breaks packaging on 18.04

Can you switch to boost/filesystem instead?

Comment (arntanguy) [reply src/panda.cpp]: Too bad I used `std::filesystem` on purpose :P
Fixed.

**Pull request merged**